README.md

```diff
@@ -190,6 +190,12 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
     cargo install wrangler
     ```
 
+    Installing wrangler on linux requires some [OpenSSL-related packages](https://docs.rs/openssl/0.10.24/openssl/#automatic) to be installed. If you don't want to deal with this, you can use vendored OpenSSL.
+
+    ```
+    cargo install wrangler --features vendored-openssl
```

Comment:
do we have option to do this w/npm?

Reply:
this isn't necessary for npm, npm distributes built binaries that already utilize vendored openssl for linux.